cmd/backup-operator/README.md

```diff
@@ -10,15 +10,12 @@ Try out etcd backup operator by running it on Kubernetes and then create a `Etcd
 >Note: The demo uses the `default` namespace.
 
 Prerequisites: 
-* access to a Kubernetes environment.
-* see [Instruction][etcd_cluster_deploy] to deploy an etcd cluster. 
+* `example-etcd-cluster` is deployed. See [instructions][etcd_cluster_deploy] to how to deploy it.
```

Comment:
"an running etcd cluster named `example-etcd-cluster`. See [instructions][etcd_cluster_deploy] "? 
No need to add phase `how to deploy it`.  it is already implied.

Reply:
> No need to add phase how to deploy it
We need the phase. That's what I have suggested in https://github.com/coreos/etcd-operator/pull/1609#discussion_r148403332. It is dismissed and should be fixed